acados/ocp_nlp/ocp_nlp_cost_common.h

```diff
@@ -59,6 +59,8 @@ typedef struct
     int (*memory_calculate_size)(void *config, void *dims, void *opts);
     struct blasfeo_dvec *(*memory_get_grad_ptr)(void *memory);
     void (*memory_set_ux_ptr)(struct blasfeo_dvec *ux, void *memory);
+    void (*memory_set_z_ptr)(struct blasfeo_dvec *z, void *memory);
```

Comment:
The name `z` is already taken for the linear cost of the slack variables `s` (the quadratic cost is `Z`).

As it is coded now, this is unclear if `memory_set_z_ptr` refers to the cost `z` or the algebraic variable `z`.
The solution is to rename one of the two.
Some ideas: `z_alg` for the algebraic variable, or conversely `Zs` and `zs` for the costs of the slack variables.
Other ideas?

Reply:
addressed here: 2504e0f (z -> z_alg, for the time being)

Reply:
great thanks :)